src/Console/ConfigurationResolver.php

```diff
@@ -459,6 +459,45 @@ private function resolveFixers()
     {
         $this->fixers = $this->fixerFactory->useRuleSet($this->ruleSet)->getFixers();
 
+        // test if there are conflicting fixers configured
+        $conflictMap = $this->getFixersConflictMap();
```

Comment:
I would put it into fixer factory.
One could use fixer factory without configuration resolver, and with handling conflicts in resolver he will be able to create conflicting fixers array.

What do you think?